python/docs/examples/time_centric.qmd

```diff
@@ -0,0 +1,209 @@
+---
```

Comment:
So we can't use notebooks anymore? I guess that's OK for our current uses (just making docs), but it was nice for examples to have a notebook people could copy. Either way, not a blocker.

Reply:
you can use notebooks, but you should save the output in the notebook to the repo.

If you want it to generate dynamically on each docs run, then use markdown.